tests/tests.py

```diff
@@ -0,0 +1,15 @@
+from hmtredishelp import RedisConn
+import time
+import sys, logging
+import unittest
+
+class HMTRedisHelpTests(unittest.TestCase):
+    def test_redis_connection(self):
```

Comment:
Good start.. some other things you can prob add to this down the road:

- Ensure that the generic "redis" functions work
- Ensure that RedisConn uses the right connection approach (ie sentinel vs redis-strict)
- Make sure it connects to the right redis endpoint params

Reply:
Thank you for feedback!